modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__.soy

```diff
@@ -32,12 +32,13 @@
 	</div>
 {/template}
 #elseif (${liferayVersion.startsWith("7.1")})
+#parse ("definitions.vm")
```

Comment:
Can you put this at the top of the file to be consistent with the other ones?

Reply:
Yes, I can. However, the variable set on _definitions.vm_ is only necessary on 7.1 template, because of that I added this line where it's. Should I move it to the first line?

Reply:
I'm move it to the top of the line so that we can be consistent.

Reply:
No problem, Greg. Next time, I'll prioritize the consistence instead of other things. 😉